### DISCUSSION TITLE
End Opening Date

### DISCUSSION THREAD

**Discussion by GinnyEvilShadow** [Stardust-Labs-MC/StardustMC#2] [Ideas]
I'm not sure if it's possible to change, beit so close to the date, but if possible, may I suggest to have the End open on the 27th instead of the 28th? I say this because the 28th is a Monday lots of people(including myself) have already started school/classes(or will be in the next week or two) and would be unable to be there for the opening. The 27th is a Sunday which most people should be able to play on, at least much moreso than Monday.

If you want it to be later than may I suggest September 2nd, as that falls on a Saturday and would be perfect for an event such as this.

Also small note, could we get an eta on what **time** the End will open? Preferably translated to multiple timezones? This way players can prepare their schedules accordingly.

**Comment by CodingPanda0623** [comment]
I would also like the end to open while I'm not at school.

**Comment by aisukuma** [comment]
We will be changing the date to the 27th
